doc/test.txt

```diff
@@ -626,6 +626,7 @@ You may find yourself specifying certain options for your test runners in
 certain situations. You can configure your preferred options with
 >
   let g:test#ruby#minitest#options = '--verbose'
+  let g:test#go#gotest#test_verbose = 1
```

Comment:
Instead of introducing a new variable for something that vim-test already offers, I'd rather we add an example of in the docs.

```suggestion
  let g:test#go#gotest#options = '-v'
```

Reply:
Thank you for your suggestion